README.md

```diff
@@ -1,6 +1,55 @@
-# Perfeed
-Personalized, AI assistant that runs with your engineering teams
+# 🚀 Perfeed
+A personalized AI assistant tailored to streamline engineering team workflows.
 
-# How to Use
-1. copy perfeed/settings/.secrets_template.toml file to .secrets.toml and fill in the values.
-2. Optional, update the values in perfeed/settings/configs.toml if you'd like to change the default configurations.
\ No newline at end of file
+## 🌟 Overview
+Perfeed is an intelligent AI assistant designed to integrate seamlessly with engineering teams, making tasks like reviewing faster and more efficient. Struggling with a heavy workload and time-consuming reviews? Perfeed is here to help!
+
+## ⚙️ Getting Started
+
+### Prerequisites
+
+1. **Python**  
+   - Install Python 3.10 or higher.  
+   - **Recommendation**: Use a [Conda virtual environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) for managing Python versions.
+
+2. **Poetry**  
+   - Install **Poetry 1.8.3**, the package manager used to handle Python dependencies. Dependencies are listed in the `pyproject.toml` file.  
```

Comment:
Does poetry version matter?

Reply:
No, but I think the version is the latest one.